README.md

```diff
@@ -123,11 +123,81 @@ pprint(feature_vector)
 }
 ```
 
+## Roadmap
```

Comment:
Any particular reason you're moving the roadmap to Github? I don't mind, just wondering why.